src/MongoDB.Bson/Serialization/Serializers/BsonClassMapSerializer.cs

```diff
@@ -54,6 +78,9 @@ public BsonClassMapSerializer(BsonClassMap classMap)
             }
 
             _classMap = classMap;
+#if NETSTANDARD1_5
+            CheckForISupportInitializeInterface(out _beginInitMethodInfo, out _endInitMethodInfo);
```

Comment:
Do the work of finding the method infos only once.

The call to Invoke will still be per deserialized document.